docs/pangeo-showcase.rst

```diff
@@ -28,32 +28,32 @@ Spring 2023 Showcase
      - Title
    * - 2023-02-01 4PM EST
      - Tom Nicholas, Columbia University 
-     - Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science
+     - `Xarray-Datatree: Hierarchical Data Structures for Multi-Model Science <https://discourse.pangeo.io/t/wednesday-february-1st-xarray-datatree-hierarchical-data-structures-for-multi-model-science/3207>`_ |Nicholas DOI Badge|
    * - 2023-02-08 12PM EST
-     - Alex Kerney, Gulf of Maine Research Institute 
+     - `Alex Kerney, Gulf of Maine Research Institute <https://discourse.pangeo.io/t/wednesday-february-8th-mental-health-in-the-geosciences/3211>`_ |Kerney DOI Badge|
```

Comment:
For consistency, the link to Discourse should be on the title of the talk, rather than the speaker name.
```suggestion
     - Alex Kerney, Gulf of Maine Research Institute
```